### PR TITLE
TBG Macros: Add Restart Dir

### PR DESCRIPTION
**Pull request by ax3l**
That flag is needed pretty often and it should be documented at `doc/TBG_macros.cfg` too that it needs to point to `<run>/simOutput/checkpoints`.

Wiki article, as linked some lines above in that file: http://git.io/PToFYg

### CODE REVIEW

File: doc/TBG_macros.cfg [hunk 1]
```diff
@@ -186,7 +186,8 @@ TBG_checkpoints="--checkpoints 1000"
 
 # Restart the simulation from checkpoints created using TBG_checkpoints
 TBG_restart="--restart --restart-step 1000"
-
+# To restart in a new run directory point to the old run where to start from
+#   --restart-directory /path/to/simOutput
```
Comment (ax3l) [inline doc/TBG_macros.cfg]: wrong, needs to be
```/path/to/simOutput/checkpoints```

Comment (PrometheusPi) [inline doc/TBG_macros.cfg]: Why are you commenting in such a way on your own pull request?

Comment (ax3l) [inline doc/TBG_macros.cfg]: So I don't forget to update it when I have the time.

(Poor mans multi tasking.)

**Comment by ax3l** [comment]
Updated :)
Sorry, I was between testing restarts and preparing a meeting when I realized that pull needs an update.

File: doc/TBG_macros.cfg [hunk 2]
```diff
@@ -186,7 +186,8 @@ TBG_checkpoints="--checkpoints 1000"
 
 # Restart the simulation from checkpoints created using TBG_checkpoints
 TBG_restart="--restart --restart-step 1000"
```
Comment (PrometheusPi) [inline doc/TBG_macros.cfg]: Could you put `--restart-step 1000` as an optional command line flag to the comment below?

Comment (ax3l) [inline doc/TBG_macros.cfg]: have you tested that it still works and finds the *latest* checkpoint automatically?

**Comment by ax3l** [comment]
I just realized: Is `--radiation_e.restart` still necessary or is it already unified?

File: doc/TBG_macros.cfg (hunk 2 above)
Comment (PrometheusPi) [inline doc/TBG_macros.cfg]: I used the automated restart point to test the *holy moly bug* #688 - it works fine.

**Comment by PrometheusPi** [comment]
The is no more `--radiation_e.restart` anymore. See [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/cf93d997ba4e8adc78a0cbb9d65ddc65e19bf719/src/picongpu/include/plugins/radiation/Radiation.hpp#L191-L202)

**Comment by ax3l** [comment]
That's great! I updated the macros and the according [wiki page](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Restarting-a-simulation/_compare/7d248368257aa3b755582953ca8c1b45d3161aae...fae9c6d4800e5ce2e90d908c3ea7723b8a91a53a) to reflect that awesomeness.

File: doc/TBG_macros.cfg (hunk 2 above)
Comment (ax3l) [inline doc/TBG_macros.cfg]: cool. but I think the description of that says it's optional. so I can rather move it down in a comment.

Comment (PrometheusPi) [inline doc/TBG_macros.cfg]: :+1: that's what I meant :smile:

**Pull request merged**